development/tools/jobs/kyma/kyma_integration_test.go

```diff
@@ -84,16 +84,6 @@ func TestKymaIntegrationJobsPresubmit(t *testing.T) {
 			},
 			expJobImage: tester.ImageBootstrapHelm20181121,
 		},
-		"Should contains the gke-central job": {
```

Comment:
I do not think we should remove the test for GKE Central presubmit job.

Reply:
Done!